lib/mini_profiler/profiler.rb

```diff
@@ -76,6 +76,17 @@ def initialize(app, config = nil)
         @config.storage_instance = @config.storage.new(@config.storage_options)
       end
       @storage = @config.storage_instance
+      if defined?(ActiveSupport)
+        ActiveSupport::Notifications.subscribe "sql.active_record" do |*args|
+          event = ActiveSupport::Notifications::Event.new *args
+          if event.payload[:cached]
```

Comment:
the fifth argument to subscribe is the hash payload, no need to create a new event object here I think?